Makefile

```diff
@@ -26,14 +26,16 @@ generate:
 test:
 	ginkgo -v -r
 
-test-acceptance: linux
-	docker-compose up --no-start
-	docker-compose start
+test-acceptance: docker-compose
 	go run cmd/stolon-pgbouncer-acceptance/main.go
 
 clean:
 	rm -rvf $(PROG) $(PROG:%=%.linux_amd64)
 
+docker-compose: linux
```

Comment:
We don't need the acceptance binary, so can get away with `bin/stolon-pgbouncer.linux_amd64` as the dependency here

Reply:
Yeah this is a very good point. Building the acceptance binary doubles the startup time, at least on my machine, so I've dropped that as suggested.